README.md

```diff
@@ -4,19 +4,79 @@ The HuBMAP Search API is a thin wrapper of the Elasticsearch. It handles data in
 
 The API documentation is available on SmartAPI at https://smart-api.info/ui/7aaf02b838022d564da776b03f357158
 
+## Working with submodule
+
+This repository relies on the [search-adaptor](https://github.com/dbmi-pitt/search-adaptor) as a submodule to function. The file `.gitmodules` contains the configuration for the URL and specific branch of the submodule that is to be used. Once you already have cloned this repository and switched to the target branch, to load the latest `search-adaptor` submodule:
+
+```
+git submodule update --init --remote
```

Comment:
We can add the config option to the README, the reason we have the line `git submodule update --init --remote` is so that the user can fetch the submodule after the clone is they did not perform `git clone ... --recursive`